How-to-Install-SCF.md

```diff
@@ -143,58 +113,97 @@ To install SCF
 
    We make use of it only because the example done here is based on the vagrant setup. Note that
    the storageclass `apiVersion` used in the manifest should either be `storage.k8s.io/v1beta1` (for
-   kubernetes 1.5.x) or `v1` (for kubernetes 1.6.x)
+   kubernetes 1.5.x) or `storage.k8s.io/v1` (for kubernetes 1.6.x)
+
+   Use kubectl to check your kubernetes server version:
    ```
-   STORAGECLASS=${STORAGECLASS:-hostpath}   
-   KUBE_MINOR_VERSION=$(kubectl version --short | grep -oE "Server Version: v1\.[0-9]+" |    sed 's/Server Version: v1\.//g')
-   if [[ $KUBE_MINOR_VERSION -eq 5 ]]; then
-       APIVERSION=storage.k8s.io/v1beta1
-   elif [[ $KUBE_MINOR_VERSION -eq 6 ]]; then
-       APIVERSION=storage.k8s.io/v1
-   fi
-   kubectl get storageclass ${STORAGECLASS} 2>/dev/null || {
-       sed kube/uaa/kube-test/storage-class-host-path.yml \
-           -e "s/^  name:.*/  name: ${STORAGECLASS}/" \
-           -e "s@^apiVersion:.*@apiVersion: ${APIVERSION}@g" | \
-       kubectl create -f -
-   }
+   kubectl version --short | grep "Server Version"
    ```
 
-* Save all choices to environment variables, as demonstrated in the example scripts associated with the individual points above.
-  These are used in the coming commands.
+   For kubernetes 1.5.x:
+   ```
+   cat > storage-class-host-path.yaml <<END
+   ---
+   kind: StorageClass
+   apiVersion: storage.k8s.io/v1beta1
+   metadata:
+     name: hostpath
+   provisioner: kubernetes.io/host-path
+   parameters:
+     path: /tmp
```

Comment:
I'm pretty sure the `parameters` here doesn't actually do anything. (It was an attempt to feed it things, but it wasn't actually used.)